docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -419,14 +419,109 @@ As discussed in treatment guideline reviews, in general triplet regimens are pre
      - High
      - VRd: bortezomib + lenalidomide + dexamethasone
 
-First-line treatment
-^^^^^^^^^^^^^^^^^^^^
+Treatment Modeling Strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Second-line treatment
-^^^^^^^^^^^^^^^^^^^^
+**Scope**:
+
+This modeling strategy allows us to track the proportion of MM/RRMM simulants that receive an isatuxamib-containing treatment. It also allows for coherent assignment of isatuxamib-containing treatments at the simulant level so that any given simulant is not assigned an isatuxamib-containing treatment an illogical amount of times, which will reduce noise in our estimations.
+
+This model strategy does NOT consider:
+
+- Differences between isatuxamib-containing treatment regimens (ex: isatuxamib + pomalidomide + dexamethasone versus isatuxamib + carfilzomib + dexamethasone)
```

Comment:
You may ignore the differences of efficacy for isa-containing regimens, but IsaPd is recommended to be used at 3rd line or later while IsaKd is recommended to be used at 2nd line or later.